cruise.umple/src/UmpleInternalParser_CodeParserHandlers.ump

```diff
@@ -29,39 +37,69 @@ class UseStatementParserAction{
   depend cruise.umple.parser.Token;
   
   isA ParserAction;
-  
+  UmpleModel umpleModel;
+ 
   public void onSuccess(Token token, ParserDataPackage data)
   {
+  /*
+  we can filter the use statement by checking if ends with .ump 
+  this is simpler and needs few modification
+  
+  the solution 2: if no file, save the mixset name for later use .
+  this can be done by do some changes to the file .ump directly. Or, we can make the file throw a null pointer exception and hendel it in UseStatementAction
+  in .ump
+  
+  */
+ 
     String value = token.getValue("use");
     String fileName="";
-		String path = "";    
+	String path = "";    
     int index =-1;
     if (data!=null && data.getFullFileAddress()!=null)    index = data.getFullFileAddress().lastIndexOf("/") ;
     if (index==-1) index = data.getFullFileAddress().lastIndexOf("\\");  
     if (index!=-1){
     	path = data.getFullFileAddress().substring(0,index+1 );
     }
     
-    synchronized(data.getHasParsed())
+synchronized(data.getHasParsed())
```

Comment:
No need to take out the indentation here

Reply:
This happens because I added some code, then I removed it.